dll/win32/shell32/CShellLink.cpp

```diff
@@ -2224,7 +2224,7 @@ static BOOL HACKISH_PathResolve(
 
     if (!Success)
     {
-        ERR("SearchPathW(pszPath = '%S') failed\n", pszPath);
+        ERR("SearchPathW(pszPath = '%S') failed. Error code: %d\n", pszPath, GetLastError());
```

Comment:
This one does not add anything, it just makes analysing harder.
If there were no previous items, GetLastError is from the api called before this one,
otherwise it will just repeat the previous error.

Reply:
What other solution would you think for this then?